### PR TITLE
Created the landing page and the pop-up menu

### PR DESCRIPTION
**Pull request by rolandfbk**
The landing page and the pop-up menu have been created by replicating the design of template 5 of the Microverse-Student-Project-1 on figma.com. As required, I tried to stick to the design as much as possible.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                    <img src="img/image_geometry_1.svg" alt="">
+                    <a href="#" onclick="showMenu()"><img class="menu-button" src="img/ic_menu.svg" alt="Menu Buttom"></a>
+                </nav>
+            </header>
+
+            <section class="headline">
+                <div class="title set">
+                    <h1>Roland<br> Ossisa Yuma</h1>
+                </div>
+                <div class="content set">
```
Comment (Donard97) [inline index.html:24]: - [ ] Well done on adding the first section of the portfolio but kindle stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9). Keep rocking 💪 

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                    <p class="description">
+                        Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                        If you like what you see and have a project you need coded, don’t hesitate and contact me.
+                    </p>
+                    <button class="start-button">
+                        Start collaboration
+                    </button>
+                </div>
+                <div class="content-bottom">
+                    <img class="f-img" src="img/image_geometry_2.svg" alt="">
```
Comment (Donard97) [inline index.html:34]: - [ ] Good job on adding the button and styling it but kindly stick as much as you can to  [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9).

Here you have a screenshot from your project:

image

Here you have a screenshot from the design:

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                <div class="content-menu">
+                    <ul>
+                        <li>Hello</li>
+                        <li>Portfolio</li>
+                        <li>About</li>
+                        <li>Contact</li>
+                    </ul>
+                </div>
+                <div class="content-bottom">
+                    <img class="f-img" src="img/image_geometry_menu_2.svg" alt="">
```
Comment (Donard97) [inline index.html:57]: - [ ] Well done on adding the image but this is a background image so kindly change it to the background and not the img tag. Keep coding 💪

File: index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        <title>Roland Portfolio</title>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+
+    <body>
+        <div class="wrapper" id="wrapper">
+            <header class="header">
+                <nav class="nav">
+                    <img src="img/image_geometry_1.svg" alt="">
```
Comment (Donard97) [inline index.html:15]: - [ ] Well done on adding the image but this is a background image so kindly change it to the background and not the img tag. Keep coding 💪

**Review by Donard97 — CHANGES_REQUESTED** [review]
## Hello @rolandfbk , 👋  

You did a tremendous job so far! 💯 👏 

There are some issues that you still need to work on to go to the next project but you are almost there! 💪 

133678845-1b015b27-e9bc-4784-a316-bc699db8237a

## Highlights 🏁 

- There is no linter error ✅ 
- Code is clean ✅ 
- Readme file is professional ✅ 

## Required Changes ♻️
- [ ] Kindly add the .hintrc file for linters following linters configuration [repository](https://github.com/microverseinc/linters-config/tree/master/html-css)
- [ ] I suggest you in the future to do more commits because it is a really good habit and it is professional.
- [ ] Please check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by aimemalaika — APPROVED** [review]
Hi @rolandfbk , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**